src/com/mambu/apisdk/util/RequestExecutorImpl.java

```diff
@@ -139,21 +141,22 @@ public String executeRequest(String urlString, ParamsMap params, Method method,
 		} catch (IOException e) {
 			LOGGER.warning("IOException: message= " + e.getMessage());
 			throw new MambuApiException(e);
+		} finally {
+			httpClient.getConnectionManager().shutdown();
```

Comment:
I believe this only releases resources for this specific instance of the HttpClient (which it is about to go out of scope). I don't  think we have any code to ensure any connection persistence in the current implementation of the RequestExecutor in the wrapper.